server/schema/postgres.sql

```diff
@@ -25,10 +25,12 @@ CREATE TABLE IF NOT EXISTS actions (
 	id uuid PRIMARY KEY DEFAULT gen_random_uuid(),
 	organization_id uuid NOT NULL REFERENCES organizations(id) ON DELETE CASCADE ON UPDATE CASCADE,
 	name text NOT NULL,
-	type text NOT NULL,
+	type_name text,
```

Comment:
Why not directly reference the action type by its id instead of its name? This prevents problems when multiple action type with the same names exist. You could also skip the version information and fetch name and version with a join from the `action_types` on getting a list of actions.